tests/vmi_kernel_boot_test.go

```diff
@@ -122,4 +122,60 @@ var _ = Describe("[sig-compute]VMI with external kernel boot", func() {
 			Expect(err.Error()).To(ContainSubstring("denied the request: spec.domain.firmware.kernelBoot.container must be defined with at least one of the following: kernelPath, initrdPath"))
 		})
 	})
+
+	Context("with external alpine-based kernel only", func() {
+		It("ensure successful boot", func() {
+			vmi := utils.GetVMIKernelBoot()
+			// Remove initrd path from vmi spec
+			kernelBoot := vmi.Spec.Domain.Firmware.KernelBoot
+			kernelBoot.Container.InitrdPath = ""
```

Comment:
nit: you can add the following function under `Context("with external alpine-based kernel only", func() {`:
```go
getVMIKernelBoot := func() *VirtualMachineInstance {
	vmi := utils.GetVMIKernelBoot()
	// Remove initrd path from vmi spec
	kernelBoot := vmi.Spec.Domain.Firmware.KernelBoot
	kernelBoot.Container.InitrdPath = ""
}
```

That way anyone writing tests in these context wouldn't have to worry about removing initrd path + code wouldn't be duplicated.